ocs_ci/resiliency/network_faults.py

```diff
@@ -0,0 +1,246 @@
+import time
+import random
+import logging
+from ocs_ci.ocs import ocp
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.ocs.platform_nodes import PlatformNodesFactory
+
+
+log = logging.getLogger(__name__)
+
+
+class NetworkFaults(PlatformNodesFactory):
+    """
+    A class to inject and remove various network faults on OpenShift cluster nodes
+    using the 'tc' Linux command via 'oc debug'.
+    """
+
+    def __init__(self, nodes, interface_type="default", duration=30):
+        """
+        Initializes the NetworkFaults object.
+
+        Args:
+            nodes (list): List of node objects.
+            interface_type (str): Type of interface to use ("default" or "ovn").
+            duration (int): Duration in seconds to keep the fault active.
+        """
+        super().__init__()
+        self.nodes = nodes
+        self.duration = duration
+        self.ocp_obj = ocp.OCP()
+        self.interface = self._get_node_network_interface(
```

Comment:
shouldn't we allow selecting multiple interfaces?

Reply:
Added multiple network selection logic

Reply:
Added multiple interfaces